charts/fybrik/files/taxonomy/datacatalog.json

```diff
@@ -1,6 +1,48 @@
 {
   "title": "datacatalog.json",
   "definitions": {
+    "CreateAssetRequest": {
+      "type": "object",
+      "required": [
+        "destinationCatalogID",
+        "resourceMetadata",
+        "resourceDetails"
+      ],
+      "properties": {
+        "credentials": {
+          "description": "This optional field has the vault plugin path where the destination data credentials will be stored as kubernetes secrets",
+          "type": "string"
+        },
+        "destinationAssetID": {
+          "description": "This is an optional field provided to give information about the asset id to be used for the created asset.",
```

Comment:
Simplify: Asset ID to be used for the created asset.

Reply:
Indeed, you don't have to explicitly specify that a field is optional, this info is auto-generated. 
Interesting, that there is no documentation to the `resourceDetails ` and `resourceMetadata`.
![image](https://user-images.githubusercontent.com/1816654/155897225-2e68cf70-6231-463d-b22e-038ffe113a2c.png)

Reply:
I agree with @shlomitk1 , you don't have to write "Tis optional field has ... ", just "vault plugin path where the destination data credentials will be stored as kubernetes secrets", see as an example K8s documentation - https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/deployment-v1/

Reply:
Fixed. Please check. Thanks.

Reply:
Regarding documentation for `resourceDetails` and `resourceMetadata` objects, I think this issue is due to the openapigenerator tool not parsing the `description` field for non-primitive objects (which are defined by using `$ref ` field in the json schema). However, there is a description shown once we click the hyperlinks corresponding to `resourceDetails` and `resourceMetadata` in the above table. 
To resolve this issue fully, we may need deeper investigation on this and it may require more effort in order to understand the tool better. So,  I am not resolving this issue for now. Thanks.